test/jest/__mock__/stripesComponents.mock.js

```diff
@@ -0,0 +1,10 @@
+jest.mock('@folio/stripes/components', () => ({
+  ...jest.requireActual('@folio/stripes/components'),
+  KeyValue: jest.fn(({ label, children }) => (
```

Comment:
I've recommend to delete this two mocked components here, because we don't ever test them in this module